english/marshmallow-attack/make-smores/make-smores.md

```diff
@@ -0,0 +1,7 @@
+Mmmmm! S'mores are tasty!  You can't wait to take a bite of the gooey chocolately crispy goodness!
+
+But wait -- do you have chocolate and graham crackers?
+
+What? No Chocolate? [Look in your pockets]  (look-for-chocolate/look-for-chocolate.md)
+
+OMG!  No chocolate AND no graham crackers [Start to cry] (/start-to-cry/start-to-cry.md)
```

Comment:
The slash at the beginning is causing this to go to the root of the repository first, but it looks to me like `start-to-cry` is within the current directory, so you could remove the slash at the beginning. (And the capitalization needs to be consistent.)